source/MaterialXGenGlsl/GlslShaderGenerator.cpp

```diff
@@ -740,7 +740,7 @@ void GlslShaderGenerator::emitVariable(const Shader::Variable& variable, const s
     if (variable.type == Type::FILENAME)
     {
         // Samplers must always be uniforms
-        shader.addLine("uniform sampler2D " + variable.name);
+        shader.addStr("uniform sampler2D " + variable.name);
```

Comment:
Note to Niklas, the impetus for this is to make all generators output only the variables without any post formatting. Then post-formatting can be done for a given block or per an individual variable emit.

Reply:
Ah, thanks for the clarification. Now I understand the change better :)